lib/jsonapi_errorable.rb

```diff
@@ -55,6 +55,10 @@ def default_exception_handler
     self.class.default_exception_handler
   end
 
+  def registered_exception?(e)
```

Comment:
I tested it [here](https://github.com/BjornMelgaard/ngrx-todolist/blob/with_jsonapi_suite/app/controllers/concerns/jsonapi_errors.rb#L33) )